src/rules/correlation.utils.ts

```diff
@@ -25,3 +31,32 @@ export function replaceCorrelatedValues({
     value: varName,
   })
 }
+
+function replaceAllTextMatches(
```

Comment:
Move function here from `shared.ts` because it's the only place where it's used.